.github/interop/runner.patch

```diff
@@ -295,3 +321,34 @@ index 2741a0d..a8d1e14 100644
  ]
  
  MEASUREMENTS = [
+diff --git a/trace.py b/trace.py
+index 215f25a..00759d4 100644
+--- a/trace.py
++++ b/trace.py
+@@ -125,10 +125,25 @@ class TraceAnalyzer:
+                 if layer.layer_name == "quic" and not hasattr(
+                     layer, "long_packet_type"
+                 ):
+-                    layer.sniff_time = packet.sniff_time
+                     packets.append(layer)
+         return packets
+ 
++    def get_start_end(self, direction: Direction = Direction.ALL) -> (int, int):
++        """ Get start and end time, one or both directions. """
++        start = 0
++        end = 0
++        for packet in self._get_packets(
++            self._get_direction_filter(direction) + "quic.header_form==0"
++        ):
++            for layer in packet.layers:
++                if layer.layer_name == "quic" and not hasattr(
++                    layer, "long_packet_type"
++                ):
```

Comment:
can you explain why we dont want "long_packet_type"

Reply:
i was just using what was there from before. this is mostly a temporary fix until it's fixed upstream